nservicebus/hosting/nservicebus-host/installation.md

```diff
@@ -20,83 +32,103 @@ NServiceBus.Host.exe [/install [/serviceName]
 [/sideBySide]
 [/startManually]
 [/username]
-[/password]]
-[/uninstall [/serviceName]
-[/sidebyside]
-[/instance:Instance Name ]
+[/password]
 ```
 
-To retrieve this list, run the following at the command line:
+Here is an example of the `/install` command line:
 
 ```dos
-NServiceBus.Host.exe /?
+NServiceBus.Host.exe /install 
+/serviceName:"MyPublisher"
+/displayName:"My Publisher Service"
+/description:"Service for publishing event messages"
+/endpointConfigurationType:"QualifiedNameSpace.EndpointConfigType, AssemblyName"
+/username:"corp\serviceuser"
+/password:"p@ssw0rd!" NServiceBus.Production
 ```
 
-To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. This is different from what is displayed in the Windows Service Manager.
+
+### ServiceName
+
+To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. By default, the name of the service is the name of the endpoint and the endpoint name is the namespace of the endpoint configuration class.
+
+Note: The Windows Service name is different from what is displayed in the Windows Service Manager. The Windows Service Manager shows the *DisplayName*.
+
+
+### DisplayName
 
 To set the name of the Windows Service as it is displayed in the Windows Service Manager, specify `/displayName:ServiceDisplayName`.
 
-If `/displayName` is not specified, but `/serviceName` is, the display name does not become what was passed in the `/serviceName`, but rather the default described above.
+If `/displayName` is not specified, but `/serviceName` is, the display name does not become what was passed in the `/serviceName`, but the default *ServiceName* as described above is used.
+
+
+### Description
 
 To set the description shown in the Windows Service Manager, specify
 `/description:DescriptionOfService`.
 
-To install multiple instances of the same service by providing each a different instance name, use the 'instance' flag. For example: `/instance:Instance5`.
+
+### EndpointConfigurationType
+
+Specify the type implementing `IConfigureThisEndpoint` that should be used by using the `/endpointConfigurationType` parameter. The provided value needs to be the [assembly qualified type name](https://msdn.microsoft.com/en-us/library/system.type.assemblyqualifiedname(v=vs.110).aspx).
+
+
+### EndpointName
+
+Configures the name of the endpoint. By default, the endpoint name is the namespace of the endpoint configuration class.
+
+
+### ScannedAssemblies
+
+Configures NServiceBus to scan only the specified assemblies.
```

Comment:
need to dig deeper into that argument since from a first glance at the code it looks like this parameter can be used repeatedly.